app/models/resource.py

```diff
@@ -0,0 +1,30 @@
+from .. import db
+
+
+class Resource(db.Model):
+    __tablename__ = 'resources'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64))
+    address = db.relationship('Address', backref='resource',
+                              lazy='dynamic', uselist=False)
+    description = db.Column(db.Text)
+    website = db.Column(db.Text)
+    reviews = db.relationship('Review', backref='resource',
```

Comment:
I think this should stay as `dynamic`. It's a one-to-many field that could possibly grow to hold a large amount of text fields that we might not want to be loaded in certain views (e.g. search results)?